HelpSource/Classes/Object.schelp

```diff
@@ -692,8 +692,129 @@ code::
 method::dump
 
 Print a detailed low level representation of the receiver to the post window.
+Except for the link::Classes/List:: class, this method is not overridden in other classes. Any of the MetaClasses, Classes and Instances can be used by this method.
```

Comment:
suggestions follow.

Any object understands this method (this means metaclasses, classes and their instances). Note that in link::Classes/List:: class, this method dumps the list's internal array.

Reply:
Thanks, done!